internal/operators/ocs/ocs_operator.go

```diff
@@ -142,21 +145,20 @@ func (o *operator) GetHostRequirements(_ context.Context, cluster *common.Cluste
 		// for each disk ocs requires 2 CPUs and 5 GiB RAM
 		if host.Role == models.HostRoleMaster || host.Role == models.HostRoleAutoAssign {
 			return &models.ClusterHostRequirementsDetails{
-				CPUCores:   CPUCompactMode + reqDisks*o.config.OCSRequiredDiskCPUCount,
-				RAMMib:     conversions.GibToMib(MemoryGiBCompactMode + reqDisks*o.config.OCSRequiredDiskRAMGiB),
-				DiskSizeGb: MinDiskSize,
+				CPUCores:   o.config.OCSPerHostCPUCompactMode + reqDisks*o.config.OCSPerDiskCPUCount,
+				RAMMib:     conversions.GibToMib(o.config.OCSPerHostMemoryGiBCompactMode + reqDisks*o.config.OCSPerDiskRAMGiB),
+				DiskSizeGb: o.config.OCSMinDiskSize,
```

Comment:
I think we should not include the `DiskSizeGb` here, as it would be added with `OCPDiskSize` .
@nbalacha  WDYT?